cluster/seq-cluster-docker-compose/compose.yaml

```diff
@@ -26,20 +31,58 @@ services:
   seq2:
     image: datalust/seq:preview
     hostname: seq2
+    depends_on:
+      pgseq:
+        condition: service_healthy
+      seq1:
+        condition: service_healthy
     networks:
       - seqclusterdemo
     volumes:
       - ./seq2data:/data
     environment: 
       - ACCEPT_EULA=Y
       - 'SEQ_METASTORE_POSTGRES_CONNECTIONSTRING=Server=pgseq;Port=5432;Database=seq;Username=postgres;Password=terminatorx;'
-      - SEQ_API_CANONICALURI=http://proxy:5666
+      - SEQ_API_CANONICALURI=http://localhost:5666
```

Comment:
The canonical URI should be one the user sees - clarifying this here, since the hostname `proxy` is internal to the Docker network.